ui/src/utils/language.ts

```diff
@@ -128,6 +129,7 @@ const ENGLISH_STRINGS: { [key in STRING]: string } = {
 
   /* OTHER */
   [STRING.CLOSE]: 'Close',
+  [STRING.LOADING_DATA]: 'Loading data',
```

Comment:
Can we display "Updating data..." when the page is loading new data in the background? I believe you mention two different loading messages in the PR description but I am only seeing one in the deploy preview.

Reply:
Yes good point, I tweaked the message to say "Updating data" when we are just updating. On initial load, we only showed the large spinner before, I added the message here as well now, but with the text "Loading data".

<img width="1132" alt="Screenshot 2023-05-08 at 10 33 11" src="https://user-images.githubusercontent.com/11680517/236776412-c9197236-c629-4d06-acbc-270f282d1a52.png">

<img width="1118" alt="Screenshot 2023-05-08 at 10 33 31" src="https://user-images.githubusercontent.com/11680517/236776485-542f5b6e-4d3b-4b06-93ff-16bb8094ea33.png">

Also did a small tweak to the error state since you last watched. If we have an error but are loading, we show the spinner, not the error message.